tel2puml/__main__.py

```diff
@@ -212,10 +230,15 @@ def generate_component_options(
             ]
         elif pv_to_puml_obj.folder_path:
             file_list = find_files(str(pv_to_puml_obj.folder_path))
+        if pv_to_puml_obj.mapping_config_file:
+            mapping_config = PVEventMappingConfig(
+                **generate_config(str(pv_to_puml_obj.mapping_config_file))
+            )
         pv_puml_options = PVPumlOptions(
             file_list=file_list,
             job_name=pv_to_puml_obj.job_name,
             group_by_job_id=pv_to_puml_obj.group_by_job,
+            mapping_config=mapping_config,
```

Comment:
again remove this and as above set pv_puml_options["mapping_config"] under the if statement